### ISSUE TITLE
Needs more validation error details

### ISSUE THREAD

**Issue by MontiniCristian** [jonasschmidt/ex_json_schema#40]
Hi, well done with this module, what about #38 PR?
It would be nice to have this feature. Also actually the error format don't let easly (from the code viewpoint) know what is going on with the validation errors, for example:

`{:error, [{"Type mismatch. Expected String but got Integer.", "#/id"}]}`

Why there's always a "#/" prefix?

**Comment by jonasschmidt** [comment]
Yes, sorry about the delay. I just released a `0.6.0-rc.1` version that should be final in terms of the public API. It's mainly missing some documentation.

The API is backward compatible and you can get the unformatted error structs by passing an option like this:
```elixir
ExJsonSchema.Validator.validate(schema, %{}, error_formatter: false)`
```

The paths are [JSON Pointer](https://tools.ietf.org/html/rfc6901)s that JSON Schema uses as well, so it makes sense to be consistent there. It's a standard, so you can find libraries to deal with resolving those pointers.

**Comment by MontiniCristian** [comment]
Thanks, great work!
I was missing the path issue, i will take a look.

**Comment by jonasschmidt** [comment]
Final version is released as `0.6.0` with some usage examples.
